metaflow/plugins/secrets/__init__.py

```diff
@@ -1,14 +1,10 @@
 import abc
+from typing import Dict
 
 
 class SecretsProvider(abc.ABC):
+    TYPE = None
+
     @abc.abstractmethod
-    def get_secret_as_env_vars(self, secret_id, options={}):
```

Comment:
Am fine with the typing but was mostly just asking to put it in the comment like the rest so that it was clear what was expected.